js/resolc/src/index.ts

```diff
@@ -93,13 +93,26 @@ export function version(): string {
 
 export async function compile(
   sources: SolcInput,
-  option: { bin?: string } = {}
+  option: {
+    optimizer?: Record<string, unknown>
+    bin?: string
+  } = {}
 ): Promise<SolcOutput> {
+  const {
+    optimizer = {
+      mode: 'z',
```

Comment:
Should this as well as the optimizer runs be argumenta to the `compile` functions?

Reply:
not sure what you mean. if no default is specified we will use that, else what ever the user specify

Reply:
Ah yes of course, I thought it would not apply to the bin but it does. Nevermind